src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__day" data-day="1">1</div>
+      <div class="calendar__day" data-day="2">2</div>
+      <div class="calendar__day" data-day="3">3</div>
+      <div class="calendar__day" data-day="4">4</div>
+      <div class="calendar__day" data-day="5">5</div>
+      <div class="calendar__day" data-day="6">6</div>
+      <div class="calendar__day" data-day="7">7</div>
+      <div class="calendar__day" data-day="8">8</div>
+      <div class="calendar__day" data-day="9">9</div>
+      <div class="calendar__day" data-day="10">10</div>
+      <div class="calendar__day" data-day="11">11</div>
+      <div class="calendar__day" data-day="12">12</div>
+      <div class="calendar__day" data-day="13">13</div>
+      <div class="calendar__day" data-day="14">14</div>
+      <div class="calendar__day" data-day="15">15</div>
+      <div class="calendar__day" data-day="16">16</div>
+      <div class="calendar__day" data-day="17">17</div>
+      <div class="calendar__day" data-day="18">18</div>
+      <div class="calendar__day" data-day="19">19</div>
+      <div class="calendar__day" data-day="20">20</div>
+      <div class="calendar__day" data-day="21">21</div>
+      <div class="calendar__day" data-day="22">22</div>
+      <div class="calendar__day" data-day="23">23</div>
+      <div class="calendar__day" data-day="24">24</div>
+      <div class="calendar__day" data-day="25">25</div>
+      <div class="calendar__day" data-day="26">26</div>
+      <div class="calendar__day" data-day="27">27</div>
+      <div class="calendar__day" data-day="28">28</div>
+      <div class="calendar__day" data-day="29">29</div>
+      <div class="calendar__day" data-day="30">30</div>
+      <div class="calendar__day" data-day="31">31</div>
```

Comment:
The task specifies not to include numbers directly in the HTML. Consider using JavaScript or another method to dynamically generate these numbers.